src/locales/en.json

```diff
@@ -115,37 +115,37 @@
-    "content": "What content types can I search for using the Meta Search feature?",
-    "content-types": "Meta Search on {openverse} currently supports image, audio, and video search on external sites.",
+    "content": "What content types can I search for using the External Sources feature?",
+    "content-types": "External Sources on {openverse} currently supports image, audio, and video search on external sites.",
     "use": " Use ",
     "images": {
-      "title": "How does the Meta Search feature work for indexed media types?",
-      "demo-label": "Layout of Meta Search for indexed media type, like images",
+      "title": "How does the External Sources feature work for indexed media types?",
+      "demo-label": "Layout of External Sources for indexed media type, like images",
       "content": "{openverse} supports built-in search for some media types, such as images. When you enter a query on the {openverse} homepage, a page of results will be displayed. Scrolling to the bottom of the page, you have two choices:",
       "continue": "{load-more}, and continue browsing in {openverse}.",
       "not-finding": "See the section “{not-finding}” below the results.",
       "info": "The “{not-finding}” section lists other sources of CC-licensed and public domain content. Your search query will be preloaded, and if supported, you can select a {use} filter to proactively filter your results. Click on the button for the external source you’d like to search, and your results will open in a new tab."
     },
     "audio-video": {
-      "title": "How does the Meta Search feature work for non-indexed media types?",
-      "demo-label": "Layout of Meta Search for non-indexed media type, like video",
+      "title": "How does the External Sources feature work for non-indexed media types?",
+      "demo-label": "Layout of External Sources for non-indexed media type, like video",
       "content": "{openverse} does not yet support built-in search for some media types, such as videos. For these media types, click on the appropriate tab and you’ll see sources where you can find CC-licensed content.",
-      "filter": "The Filters bar on the left includes the {use} filter, which may be supported by some meta search sources. Apply these before clicking on the button for the external source you’d like to search."
+      "filter": "The Filters bar on the left includes the {use} filter, which may be supported by some external sources sources. Apply these before clicking on the button for the external source you’d like to search."
```

Comment:
We have removed filters from the external sources searches. We should probably remove this string, too, and whatever is using it.